src/elfpy/pricing_models/hyperdrive.py

```diff
@@ -118,6 +118,7 @@ def calc_in_given_out(
             out=Quantity(amount=float(Decimal(out.amount) * Decimal(time_remaining.normalized_time)), unit=out.unit),
             market_state=market_state,
             fee_percent=fee_percent,
+            # TODO: don't hardcode days to 365, initialize to term length
```

Comment:
IMO, this change should be part of this effort. We should have tests with non-365 day terms that capture the problems caused by our current normalization (again, this won't manifest on this line).

Reply:
Agreed, although IMO the test PR should not include new features so I'd say make that its own PR

Reply:
I'm okay with testing PRs that fix the problems they test; however, I'm fine with this approach in this case.